modules/zivid/captureassistant.py

```diff
@@ -0,0 +1,85 @@
+"""Contains capture assistant functions and classes."""
```

Comment:
As discussed IRL, this module should probably be called `capture_assistant`. This would reflect the convention of `point_cloud` and such.

Reply:
Done in fixup

Reply:
Done in fixup.